src/CommunityVoices/App/Website/Presentation/Module/QuoteCollection.xslt

```diff
@@ -23,6 +23,11 @@
           <xsl:with-param name="rightButtons">
               <xsl:if test="$isManager">
                 <a class="btn btn-outline-primary mr-2" href="/community-voices/quotes/new">+ Add quote</a>
+                <!-- https://stackoverflow.com/questions/1084925/input-type-file-show-only-button?page=1&tab=votes#tab-top -->
+                <form action='/community-voices/quotes/confirm' method='post' enctype='multipart/form-data' id="batchUploadForm">
+                    <input class="custom-file-input" id="file" type='file' name='file[]' multiple="" accept='.xlsx, .xls, .csv' style="display: none;"/>
```

Comment:
Should we accept Excel files?  I thought we moved on to only CSV

Reply:
@Sammidysam Good point. Will change

Reply:
Fixed